code/game/objects/items/devices/flashlight.dm

```diff
@@ -331,14 +330,15 @@
 	name = "torch"
 	desc = "A torch fashioned from some leaves and a log."
 	w_class = WEIGHT_CLASS_BULKY
-	brightness_on = 4
+	brightness_on = 6 //When on were like a lantern
 	light_color = "#FAA44B"
 	icon_state = "torch"
 	item_state = "torch"
 	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
 	light_color = LIGHT_COLOR_ORANGE
-	on_damage = 10
+	total_mass = 0.5
```

Comment:
This is too damn low, for comparison the total_mass of a tiny item is 1.25.
It should be more or less `TOTAL_MASS_NORMAL_ITEM` instead.